spec/minimax_server/server_configuration_spec.clj

```diff
@@ -0,0 +1,18 @@
+(ns minimax-server.server-configuration-spec
+  (:import [scarvill.httpserver.server ServerConfiguration HttpService])
+  (:require [speclj.core :refer :all]
+            [minimax-server.server-configuration :refer :all]))
+
+(describe "server configuration"
+
+  (it "implements ServerConfiguration"
+      (should-be-a ServerConfiguration (minimax-server-config 5000 identity)))
+
+  (it "returns port given at init"
+      (should= 5000 (.getPort (minimax-server-config 5000 identity))))
+
+  (it "returns nil for public directory"
+      (should= nil (.getPublicDirectory (minimax-server-config 5000 identity))))
+
+  (it "returns an HttpService"
+      (should-be-a HttpService (.getService (minimax-server-config 5000 identity)))))
```

Comment:
The ServerConfig is necessary to run the server. These tests were more or less just making sure that this interface was implemented. The only function available is .serve(Socket client) for the HttpService is to take a client socket and set up the IO, then handle it's request. Would you recommend that we test that this HttpService can handle a client in this test file?